apps/web/src/styles/index.css

```
@@ -333,3 +333,195 @@
+.docs-prose :not(pre) > code {
+  font-family: var(--font-mono);
+  font-size: 0.84em;
+  background-color: hsl(var(--muted) / 0.5);
+  color: hsl(var(--foreground));
+  border: 1px solid hsl(var(--border));
+  padding: 0.1em 0.35em;
+}
+
+/* Code blocks — square border, bg-muted/30 matching feature card style */
+.docs-prose pre {
+  background-color: hsl(var(--muted) / 0.3) !important;
```

Stylelint (17.7.0)

[error] Expected "0.3" to be "30%" (alpha-value-notation)

(alpha-value-notation)